provisioning/baremetal_pod.go

```diff
@@ -293,7 +321,7 @@ func createInitContainerMachineOsDownloader(info *ProvisioningInfo, setIpOptions
 	initContainer := corev1.Container{
 		Name:            "metal3-machine-os-downloader",
 		Image:           info.Images.MachineOsDownloader,
-		Command:         []string{"/usr/local/bin/get-resource.sh"},
+		Command:         []string{"/usr/local/bin/get-live-images.sh"},
```

Comment:
You need to change how `env` is built above.

Reply:
Can you please elaborate why we need to rebuild the `env`? The image URL is sent via the installer to CBO via provisioning config and we control what gets sent here.